src/Entity/ShipmentTotalWeight.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+use Ups\Entity\UnitOfMeasurement;
+
+class ShipmentTotalWeight implements NodeInterface
+{
+    private $unitOfMeasurement;
+    private $Weight;
```

Comment:
Typehint missing, not sure which one it should be.

Reply:
Not sure what your asking for me here. No other entities has type hinting with the class properties.